apps/admin/frontend/src/app.test.tsx

```diff
@@ -292,7 +295,7 @@ test('L&A (logic and accuracy) flow', async () => {
       expect.anything()
     )
   );
-  jest.advanceTimersByTime(5000);
+  advanceTimers(5);
```

Comment:
Needed to switch from plain `jest.advanceTimersByTime` to our `advanceTimers` helper to prevent warnings about missing `act` wrappings:

```
Warning: An update to SessionTimeLimitTimer inside a test was not wrapped in act(...).
```